app/Controllers/categoryController.php

```diff
@@ -41,18 +41,18 @@ public function createAction() {
 			invalidateHttpCache();
 
 			$cat_name = trim(Minz_Request::param('new-category', ''));
-			if ($cat_name == '') {
+			if ($cat_name === '') {
 				Minz_Request::bad(_t('feedback.sub.category.no_name'), $url_redirect);
 			}
 
 			$cat = new FreshRSS_Category($cat_name);
 
-			if ($catDAO->searchByName($cat->name()) != null) {
+			if ($catDAO->searchByName($cat->name()) !== null) {
```

Comment:
Many of our database functions return false during errors (though not done for `searchByName`, but it should have been done).